codespell_lib/data/dictionary.txt

```diff
@@ -17654,6 +17655,7 @@ multitute->multitude
 multivriate->multivariate
 multixsite->multisite
 multliple->multiple
+multliples->multiples
```

Comment:
Also multlipler ?

Reply:
Done and some other variants.

Reply:
Ironically, it looks like you didn't add the `multlipler` that I was suggesting? :wink: